src/App.tsx

```diff
@@ -76,11 +84,14 @@ export default class App extends React.Component<IAppProps> {
                             <div className={`app-shell platform-${platform}`}>
                                 <TitleBar icon="fas fa-tags"
                                     title={this.props.currentProject ? this.props.currentProject.name : ""}
-                                    fullName={this.props.auth.fullName}>
+                                    fullName={ApiService.getToken()? this.props.auth.fullName: ""}>
```

Comment:
I'm not sure if calling `ApiService.getToken()` is reasonable here. It was supposed to be used only in the API client, as it is where you actually need the token.

How about just checking if user is logged in in the redux store?